dds_web/database/db_utils.py

```diff
@@ -16,10 +18,19 @@ def get_facility_column_by_username(fname, column) -> (str):
     return getattr(facility, column)
 
 
-def get_facilty_projects(fid, only_id=False) -> (list):
+def get_facilty_projects(facility_id) -> (list):
     """Gets all the project for the facility ID"""
-    project_list = models.Project.query.filter_by(facility_id=fid).all()
-    return project_list if not only_id else [prj.id for prj in project_list]
+
+    # Get all projects connected to facility
+    try:
+        project_list = models.Project.query.filter_by(facility_id=facility_id).all()
+    except sqlalchemy.exc.SQLAlchemyError:
+        raise
+
+    if not project_list:
+        project_list = []
+
+    return project_list
```

Comment:
Not a big deal but you could write something like
`return project_list if project_list else []`
instead of writing a conditional first and then a return